source/framework/core/inc/TRestGDMLParser.h

```diff
@@ -2,9 +2,10 @@
 #ifndef restG4_TRestGDMLParser
 #define restG4_TRestGDMLParser
 
-#include "TGeoManager.h"
+#include <TGeoManager.h>
```

Comment:
#include "TGeoManager.h"? I am confused since typically you are doing the right thing. I am missing something? TGeoManager belongs to restsoft right?

Reply:
TGeoManager is a ROOT class.